ProcessMaker/screen-builder
# FOUR-6752 Vuex structural changes

## Issue & Reproduction Steps

Expected behavior: 
- Store are automatically loaded without manual code on `created()` components events
- The vuex folder structure follows the Vuex official guidelines per https://v3.vuex.vuejs.org/guide/structure.html

Actual behavior: 
- Store is being loaded with code on the `created()` component event
- The vuex folder structure didn't follow the Vuex guidelines per https://v3.vuex.vuejs.org/guide/structure.html

## Solution
- No need to manually "install" or register the stores per file as we saw in `vue-form-builder.vue` . We just register the stores in the main Vue file, and they are registered automatically `onLoad` 

## How to Test
Test the steps above
* Open ScreenBuilder
* Make sure that Vuex stores are registering and events are dispatching correctly to the stores.

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-6752

## Code Review Checklist
